configuration_test.go

```diff
@@ -97,16 +97,18 @@ func TestIsProjectPackage(t *testing.T) {
 }
 
 func TestStripProjectPackage(t *testing.T) {
-
-	Configure(Configuration{ProjectPackages: []string{
-		"main",
-		"star*",
-		"example.com/a",
-		"example.com/b/*",
-		"example.com/c/**",
-	}})
-
 	gopath := os.Getenv("GOPATH")
+	Configure(Configuration{
+		ProjectPackages: []string{
+			"main",
+			"star*",
+			"example.com/a",
+			"example.com/b/*",
+			"example.com/c/**",
+		},
+        SourceRoot: gopath + "/src/",
```

Comment:
Mixed spaces & tabs, maybe it's worth adding `go fmt` to CI too (or just running it locally :d)?